acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/tls/support/client/BasicClientTlsCertificateOptions.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.tests.tls.support.client;
+
+import tech.pegasys.ethsigner.core.config.tls.client.ClientTlsCertificateOptions;
+
+import java.nio.file.Path;
+
+public class BasicClientTlsCertificateOptions implements ClientTlsCertificateOptions {
```

Comment:
nit: I can see ClientTlsCertificateOptions in the code, but not in this PR - is it well named? Or is it simply a PasswordProtectedKeystore?

Reply:
`ClientTlsCertificationOptions `has been renamed from `PkcsStoreConfig`. As I believe we can technically use keystore or PEM or any other mechanism to represent client tls certificate options.

Reply:
have been renamed to BasicKeyStoreOptions